obelix_client/__init__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# This file is part of Invenio.
```

Comment:
change it to: # This file is part of Obelix.   ?
Would it be enough to create a License file and remove the header from the other files, or just keep it in __init__.py or api.py ?